components/templates/Function/index.js

```diff
@@ -3,6 +3,10 @@
 var React = require('react');
 
 var TemplateFn = React.createClass({
+  propTypes: {
+    template: React.PropTypes.string,
```

Comment:
oh rly... it's a bit problematic during the prototyping phase though. You don't know what exactly you'll need, and you want to iterate fast, not make the perfect interface just yet :P